health/health.d/tcp_listen.conf

```diff
@@ -71,12 +71,12 @@
       on: ip.tcp_syn_queue
       os: linux
    hosts: *
-  lookup: sum -60s unaligned absolute of TCPReqQFullDoCookies
+  lookup: average -60s unaligned absolute of TCPReqQFullDoCookies
    units: cookies
    every: 10s
-    warn: $this > 0
-    crit: $this > (($status == $CRITICAL) ? (0) : (60))
-   delay: up 0 down 5m multiplier 1.5 max 1h
+    warn: $this > 1
+    crit: $this > (($status == $CRITICAL) ? (0) : (5))
+   delay: up 5 down 5m multiplier 1.5 max 1h
```

Comment:
i think delay should be lookup time * n, so 10 or 20

and please update

https://github.com/netdata/netdata/blob/cdf698d71702b8934d1f0e1aaf6588b225de0fd8/health/health.d/tcp_listen.conf#L57-L68